tests/js/spec/components/dataExport.spec.jsx

```diff
@@ -36,7 +36,7 @@ describe('DataExport', function() {
       mockRouterContext(mockAuthorizedOrg)
     );
     expect(wrapper.isEmptyRender()).toBe(false);
-    expect(wrapper.text()).toBe('Export All to CSV');
+    expect(wrapper.text()).toContain('Export All to CSV');
```

Comment:
Changing these from `toBe` to `toContain` seemed to make more sense than verifying `Export All to CSVbeta` was the return value.